user-guides/README.md

```diff
@@ -38,24 +38,32 @@ Buy, mint, and sell NFTs using the Ocean Market following the guides below.
 
 Earn rewards by obtaining veOCEAN, farm yield by curating datasets, and optimize your APY by publishing Data & Algorithm NFTs on-chain. ⛓️
 
-{% content-ref url="get-started-df.md" %}
```

Comment:
All the links for the data farming are broken

Reply:
Please see comments below & use the 2nd gitbook link.
I have reviewed and it seems to be working, resolving this convo.